skills/cmux-markdown/references/commands.md

```
@@ -1,69 +1,29 @@
-### Options
-
 | Flag | Description | Default |
 |------|-------------|---------|
 | `--workspace <id\|ref\|index>` | Target workspace | `$CMUX_WORKSPACE_ID` |
 | `--surface <id\|ref\|index>` | Source surface to split from | Focused surface |
 | `--window <id\|ref>` | Target window | Current window |
 
-### Output
+## Output
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)